themes/bootstrap3/templates/RecordTab/holdingsils/electronic.phtml

```diff
@@ -23,7 +40,7 @@
             <b><?=$this->transEsc("Item Notes")?>:</b>
             <ul>
               <?php foreach ($item['item_notes'] as $item_note): ?>
-                <li><?=$this->escapeHtml($item_note) ?></li>
+                <li><?=$item_note ?></li>
```

Comment:
Was this reverted intentionally, or just a merge error?

Reply:
This is a merge error.